### PR TITLE
Added Random Delay to GUI Settings page

### PR DESCRIPTION
**Pull request by aahurtado**

### CODE REVIEW

File: pages/settingspage.py [hunk 1]
```diff
@@ -106,32 +123,29 @@ def set_data(self):
...
+                    "random_delay_start": self.random_delay_start.text(),
+                    "random_delay_stop":  self.random_delay_stop.text()}
         write_data("./data/settings.json",settings)
         self.update_settings(settings)
         QtWidgets.QMessageBox.information(self, "Phoenix Bot", "Saved Settings")
 
-    def update_settings(self,settings_data):
-        global webhook, webhook_on_browser, webhook_on_order, webhook_on_failed, browser_on_failed, dont_buy
+    def update_settings(self, settings_data):
+        global webhook, webhook_on_browser, webhook_on_order, webhook_on_failed, browser_on_failed, dont_buy, rand_delay_start, rand_delay_stop
```
Comment (Strip3s) [inline pages/settingspage.py:145]: Is this misspelled? I think it should be `random_delay_start ` & `random_delay_stop`

**Review by Strip3s — CHANGES_REQUESTED** [review]

File: pages/settingspage.py (hunk 1 above)
Comment (aahurtado) [reply pages/settingspage.py:145]: In settings.py I spelled them rand_delay_start, rand_delay_stop so it is not a bug.

Comment (Strip3s) [reply pages/settingspage.py:145]: yeah i think that's what initially threw me off. Lets make it consistent then.

Comment (aahurtado) [reply pages/settingspage.py]: Fixed!

**Review by Strip3s — DISMISSED** [review]

File: pages/settingspage.py [hunk 2]
```diff
@@ -106,32 +123,29 @@ def set_data(self):
...
-
-
-
-
-
-
+        if settings_data["random_delay_start"] != "":
+            settings.random_delay_start = settings_data["random_delay_start"]
+        if settings_data["random_delay_stop"] != "":
+            settings.random_delay_stop = settings_data["random_delay_stop"]
```
Comment (broglea) [inline pages/settingspage.py:151]: As to not potentially break people's existing settings. Can we instead make this:
```suggestion
        if settings_data.get("random_delay_start", "") != "":
            settings.random_delay_start = settings_data["random_delay_start"]
        if settings_data.get("random_delay_stop", "") != "":
            settings.random_delay_stop = settings_data["random_delay_stop"]
```

**Review by gr33k — APPROVED** [review]
Looks good

**Review by broglea — COMMENTED** [review]
One small suggestion I made with github, if folks are fine with it we can accept in UI, then i am good to approve after that. This helps with backwards compatibility of not overwriting existing settings data of people.

**Comment by broglea** [comment]
@aahurtado  In future UI changes, can we include screenshots in the PR description please? Just makes it a little easier to see the UI change :)

**Review by broglea — APPROVED** [review]
LGTM

**Review by RyeBreadd — APPROVED** [review]

**Pull request merged**